ansible-ipi-install/roles/scale-node-prep/tasks/main.yml

```diff
@@ -66,21 +66,16 @@
         - "{{ dell_nodes }}"
       register: wait_for_idrac
       until: wait_for_idrac is succeeded
-      retries: 20
+      retries: 10
       delay: 30
+      ignore_errors: true
```

Comment:
@mukrishn why are we ignoring errors here? The reason we had this task was to have the task that sets boot order wait until the iDrac was responsve.

Reply:
OK you have rescue within `20_set_director_mode.yml` ?

Reply:
yes i am ignoring here and handling them in the following tasks `20_set_director_mode.yml`. This way playbook generally wait for the iDRAC to respond(for good servers) and not stop from handling in next task.